<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CubeSettingsDialog</class>
 <widget class="QDialog" name="CubeSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>跟踪框设置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Debug/icon/boxSetting.png</normaloff>../Debug/icon/boxSetting.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>404</y>
     <width>163</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="HeightcomboBox">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>500</y>
     <width>63</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>496</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>高度：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>72</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>左上</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>经度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeftTopLonlineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeftTopLatlineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <zorder>LeftTopLatlineEdit</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>LeftTopLonlineEdit</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>72</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>右下</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>经度：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RightBottonLonlineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="RightBottonLatlineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>12</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NamelineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>12</y>
     <width>339</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>40</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>下限高度：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="HeightDownComboBox">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>40</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="HeightUpComboBox">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>40</y>
     <width>79</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>40</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>上限高度：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>162</y>
     <width>443</width>
     <height>155</height>
    </rect>
   </property>
   <property name="title">
    <string>反射率选择</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>22</y>
      <width>429</width>
      <height>123</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>选择已有的</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>218</x>
        <y>22</y>
        <width>35</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>15</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_6">
      <property name="geometry">
       <rect>
        <x>266</x>
        <y>22</y>
        <width>35</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="geometry">
       <rect>
        <x>308</x>
        <y>22</y>
        <width>37</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>25</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_8">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>22</y>
        <width>35</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>30</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_11">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>58</y>
        <width>35</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>45</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_14">
      <property name="geometry">
       <rect>
        <x>266</x>
        <y>58</y>
        <width>33</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>60</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_9">
      <property name="geometry">
       <rect>
        <x>46</x>
        <y>58</y>
        <width>39</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>35</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_15">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>58</y>
        <width>35</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>65</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_10">
      <property name="geometry">
       <rect>
        <x>94</x>
        <y>58</y>
        <width>37</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>40</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_12">
      <property name="geometry">
       <rect>
        <x>178</x>
        <y>58</y>
        <width>37</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_1">
      <property name="geometry">
       <rect>
        <x>46</x>
        <y>22</y>
        <width>39</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-5</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>94</x>
        <y>22</y>
        <width>37</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>178</x>
        <y>22</y>
        <width>33</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_13">
      <property name="geometry">
       <rect>
        <x>218</x>
        <y>58</y>
        <width>33</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>55</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>136</x>
        <y>22</y>
        <width>37</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_16">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>58</y>
        <width>37</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>70</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>全部手动添加</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>252</x>
        <y>4</y>
        <width>163</width>
        <height>89</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>36</y>
        <width>55</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>38</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>36</y>
        <width>55</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>44</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>（米）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>44</y>
     <width>33</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>（米）</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>334</y>
     <width>209</width>
     <height>59</height>
    </rect>
   </property>
   <property name="title">
    <string>回波顶高阀值</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>173</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>334</y>
     <width>225</width>
     <height>59</height>
    </rect>
   </property>
   <property name="title">
    <string>回波体积阀值</string>
   </property>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>20</y>
      <width>187</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CubeSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CubeSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
